ios/ScanbotSDKExampleReact.xcodeproj/project.pbxproj

```diff
@@ -299,18 +299,22 @@
 				TargetAttributes = {
 					00E356ED1AD99517003FC87E = {
 						CreatedOnToolsVersion = 6.2;
+						DevelopmentTeam = FRUPYT6KB3;
```

Comment:
@rutkayb I see the changes to this file are in the pull request - it would probably be better if our DevelopmentTeam ID is not visible in the repo?